### ISSUE TITLE
Data problem

### ISSUE THREAD

**Issue by tjulyz** [wwoody827/cvpr-2018-autonomous-driving-autopilot-solution#2]
Thanks for your kind share!
Can you share the dataset processing code for transferring the competition dataset to coco-like json?
I think you have use mask rcnn with json data, right?
Thanks a lot!

**Comment by wwoody827** [comment]
No,  this project does not use coco format. Please refer to official website (https://www.kaggle.com/c/cvpr-2018-autonomous-driving) for details.
